_posts/2022-08-23-getting-around-7-day-cookie.markdown

```diff
@@ -0,0 +1,95 @@
+---
+layout: epic
+title: "Hacking Around Safari's 7-day Cookie Limit"
+subtitle: What we learned while trying to fix our cookie consent banner
+date: 2022-08-23
+categories: [Safari Cookies Consent Banner]
+author: [chris]
+---
+
+Amongst the many, many things that organizations have to contend with around
+cookie consent laws is Apple's very own Safari. Did you know that Safari will
+only persist client-side cookies for 7 days? This is in support of Apple's
+[Intelligent Tracking Prevention (ITP)](https://clearcode.cc/blog/intelligent-tracking-prevention-faq),
+designed to improve user privacy.
+
+These privacy efforts are great for users, but in hand with laws like the
+[GDPR](https://gdpr-info.eu/) and [CCPA](https://oag.ca.gov/privacy/ccpa) they
+create a UX nightmare for users. Here at Artsy we've landed on a way to make
+things slightly less bad and want to share our approach.
+
+<!-- more -->
+
+Scenario: Imagine you're a resident of the EU and visit Artsy.net for the first
+time. A banner appears asking you to `Accept` or `Deny` tracking cookies from
+our site. You don't like tracking cookies, so you click the 'Deny' button and
+the banner disappears. All good, right? Nope! You visit Artsy a week later and
+again, a banner appears asking you about your preference. This happens again and
+again until you switch browsers and realize that what you were just experiencing
+was Apple's ITP in action; even though you've rejected tracking cookies, the
+cookie we've used to store your preferences is erased after 7 days,
+necessitating another interaction.
+
+While pondering how to work around this UX issue we landed on a pretty simple
+solution, prompted by a comment from a Safari Engineer we were able schedule a
+conference call with. She mentioned that the 7-day cookie limitation only
```

Comment:
This first sentence scans a bit awkwardly. I might suggest something more like: 
We thrashed around this in this vicious cycle for months until we found a simple, elegant solution thanks to a WebKit engineer's prompt (during Apple's open lab calls at WWDC). She mentioned [...]

Reply:
Updated with your suggestion 👍